week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,60 @@
+// PART-1
+
+// Clicking on the buttons should change the "theme" of the website
+
+// Blue Button
+
+let btnBlue = document.querySelector("#blueBtn");
+btnBlue.addEventListener("click", ChangeColor);
+function ChangeColor() {
+  document.querySelector(".jumbotron").style.background = "#588FBD";
+  document.querySelector(".buttons .btn-primary").style.background = "#FFA500";
+  document.querySelector(".buttons .btn-secondary").style.background = "black";
+  document.querySelector(".buttons .btn-secondary").style.color = "white";
+}
+// Orange Button
+
+let orangeBtn = document.querySelector("#orangeBtn");
+orangeBtn.addEventListener("click", ChangeColor1);
+function ChangeColor1() {
+  document.querySelector(".jumbotron").style.background = "#f0ad4e";
+  document.querySelector(".buttons .btn-primary").style.background = "#5751fd";
+  document.querySelector(".buttons .btn-secondary").style.background =
+    "#31b0d5";
+  document.querySelector(".buttons .btn-secondary").style.color = "white";
+}
+
+// Green Button
+let grnBtn = document.querySelector("#greenBtn");
+grnBtn.addEventListener("click", ChangeColor2);
+function ChangeColor2() {
+  document.querySelector(".jumbotron").style.background = "#87ca8a";
+  document.querySelector(".buttons .btn-primary").style.background = "black";
+  document.querySelector(".buttons .btn-secondary").style.background =
+    "#8c9c08";
+}
+//PART-2
+// When the submit button is pressed, it should check that all the form fields are valid:
+
+let submitBtn = document.querySelector("form .btn");
+submitBtn.addEventListener("click", CheckForm);
+function CheckForm(e) {
+  e.preventDefault();
+  let email = document.querySelector("#exampleInputEmail1");
+  let name = document.querySelector("#example-text-input");
+  let describe = document.querySelector("#exampleTextarea");
+  let form = [email, name, describe];
+  let a = !email.value.length > 0 || !email.value.includes("@");
```

Comment:
How about naming those with something that makes better sense. Then the next time you look at your code, it will be a lot easier to remember what you were doing.
eg
```
a---> isValidEmail
b--->isValidName
c--->isValidDescription
```
Considering isValidName and isValidDescription are both doing the same check, checking the number of characters of a string, how about you create a function `const hasValue =(input)=>input.value.length>0;` That way, you avoid code repetition.